code/game/machinery/computer/HolodeckControl.dm

```diff
@@ -230,6 +230,9 @@
 	for(var/obj/effect/decal/cleanable/blood/B in linkedholodeck)
 		qdel(B)
 
+	for(var/obj/effect/decal/cleanable/blooddrip/D in linkedholodeck)

```

Comment:
Could blooddrip be a subtype of cleanable/blood so you wouldn't need a separate check for them?